src/systemTags.js

```diff
@@ -0,0 +1,106 @@
+var Promise = require('promise')
+const dav = require('davclient.js')
+
+/**
+ * @class Files
+ * @classdesc
+ * <b><i> The Files class, has all the methods for your ownCloud files management.</i></b><br><br>
+ * Supported Methods are:
+ * <ul>
+ *  <li><b>Files Management</b>
```

Comment:
please adjust

Reply:
I guess we could just remove those, I expect we should be able to generate proper API docs from the JSDoc blocks